lib/tasks/import_designations.rake

```diff
@@ -29,13 +31,32 @@ namespace :stockit do
         order.country_id = Country.find_by(stockit_id: value["country_id"]).try(:id)
       end
 
-      order.status = value["status"]
+      order.state = case value["detail_type"]
+                    when "CarryOut", "Shipment"
+                      Order::SHIPMENT_STATUS_MAP[value["status"]]
+                    when "LocalOrder"
+                      Order::LOCAL_ORDER_STATUS_MAP[value["status"]]
+                    else
+                      value["status"].try(:downcase)
+                    end
+
       order.code = value["code"]
       order.detail_type = value["detail_type"]
       order.created_at = value["created_at"]
-      order.description = value["description"]
+      order.updated_at = value["updated_at"] if value["updated_at"].present?
```

Comment:
Do we need `Time.parse` here? and on other dates?

Reply:
Nope. I verified the dates. It works fine considering timezone as well.